ui/src/components/AwsProvisionerWorkerTypeTable/index.jsx

```diff
@@ -88,10 +88,18 @@ export default class AwsProvisionerWorkerTypeTable extends Component {
         <DataTable
           items={sortedWorkerTypes}
           headers={[
-            'Worker Type',
-            'Pending Tasks',
-            'Running Capacity',
-            'Pending Capacity',
+            { label: 'Worker Type', id: 'Worker Type', type: 'String' },
```

Comment:
I think we should make the id be the property key (e.g., `workerType`). The type should be whatever `typeof <value>` returns. In our case, `typeof 'some-string' will return `string`.

```
{ label: 'Worker Type', id: 'workerType', type: 'string' },
```

Reply:
Did you test that it actually works in the UI?

Reply:
Yes I tested it out and the table displays with the headers like it did before. I'll make the changes and update my PR :)

Question : What is the purpose of the `type` key, I am not sure I fully understand what it does.

Reply:
Good question. The purpose of `type` is so that we can right-align numbers in tables for better user experience. The type will help identify columns with numbers. The right alignment doesn't need to be done here. We can add that later.

Reply:
Ok, I get it now. Thank you for the response!